app/webpack/taxa/show/containers/charts_container.js

```diff
@@ -7,6 +7,8 @@ import {
   openObservationsSearch
 } from "../ducks/observations";
 
+const TermsToChart = ["Life Stage"];
```

Comment:
This is anal, but can we reserve capitalized camelcase for classes and modules and the like? I'd have this as `TERMS TO_CHART`